packages/store-sync/src/recs/syncToRecs.ts

```diff
@@ -59,9 +68,61 @@ export async function syncToRecs<TConfig extends StoreConfig = StoreConfig>({
 
   world.registerDisposer(stopSync);
 
+  const functionSelectorToResourceSelector = new Map<Hex, Hex>();
+
+  const getResourceSelector = async (functionName: string): Promise<Hex> => {
+    const functionSignature = getAbiItem({
```

Comment:
can this throw?

Reply:
No - will be undefined if function doesn't exist. I can add a custom error for that